frontend/src/components/RatingGroup/RatingGroup.tsx

```diff
@@ -1,6 +1,7 @@
 import type { RatingGroupProps } from './RatingGroup.types';
 import styles from './RatingGroup.module.css';
 import { cva } from 'class-variance-authority';
+import React, { useState } from 'react';
```

Comment:
Let's move vendor imports to the top and separate with an empty line from internal imports for better readability.
And it would be nice to be consistent and use one approach: import `React` and use all props and methods as `React.useState`, `React.Children`.....etc. or import {useState, cloneElement}